### PR TITLE
Add delegated wallets concepts - intentionally minimal doc

### PR DESCRIPTION
**Pull request by nikitaeverywhere**
Adding a minimal part to validate the diagram/flow.

### CODE REVIEW

File: docs/concepts/delegated-wallets.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+[DeCash API](/docs/api/overview) provides endpoints to create and manipulate Delegated Wallets -
+virtual multi-purpose wallets controlled by users. Delegated wallets provide the following key
+properties to the users of DeCash tokens:
+
++ DeCash API accepts transactions to Delegated Wallets and pay fees in [Algo](#TODO) on behalf of users.
++ DeCash API abstracts the [DeCash token opt-in](/docs/concepts/tokens#token-opt-in) procedure and will automatically opt Delegated Wallets in.
+
+Users can interact with Delegated Wallets in the following manner:
+
+![Delegated wallets scheme](/img/delegated-wallets.png)
```
Comment (petitnau) [inline docs/concepts/delegated-wallets.md]: I'm not sure I'm understanding this, the rest LGTM

Comment (nikitaeverywhere) [reply docs/concepts/delegated-wallets.md]: Do you mean the scheme is weird? Can you draft another one then?

Comment (nikitaeverywhere) [reply docs/concepts/delegated-wallets.md]: Here's my draft: https://drive.google.com/file/d/1ZlF3WsriTIyToAkkalnZKVeWOOoEaZE5/view?usp=sharing @petitnau

Comment (petitnau) [reply docs/concepts/delegated-wallets.md]: Sent you my draft

**Pull request merged**